exercises/practice/after-party/.docs/instructions.md

```diff
@@ -0,0 +1,50 @@
+# After Party
```

Comment:
It might be worth moving this header/paragraph to an `introduction.md`, leaving the instructions in this file and moving the hints to a hint file.

See, say, [Eliuds Eggs](https://github.com/exercism/python/tree/main/exercises/practice/eliuds-eggs/.docs) for an example of an intro vs instructions. See [Darts](https://github.com/exercism/python/tree/main/exercises/practice/darts/.docs) for an example hints file. Though ... I don't think any of the hints given here would actually apply on the Python track.